src/overlays/actors/ovl_En_Horse/z_en_horse.h

```diff
@@ -86,7 +86,13 @@ typedef enum {
 
 typedef enum {
     /* 0 */ HORSE_EPONA,
-    /* 1 */ HORSE_HNI
+    /* 1 */ HORSE_HNI,
+    /* 2 */ HORSE_TYPE_2,
+    /* 5 */ HORSE_TYPE_5 = 5,
+    /* 6 */ HORSE_TYPE_6,
+    /* 7 */ HORSE_TYPE_7,
+    /* 8 */ HORSE_TYPE_8,
+    /* 9 */ HORSE_TYPE_9
 } HorseType;
```

Comment:
it looks like this enum is meant for EnHorse#type, not as the value passed by params

the `horse_type_` should be another enum, idk if current HorseType should be renamed or what

Reply:
I'll look into it, ty